packages/react/src/progress-indicator/ProgressIndicator.test.tsx

```diff
@@ -156,110 +156,57 @@ describe('ProgressIndicator', () => {
 			};
 		}
 
-		describe('blocked status active', () => {
-			it('renders active link', () => {
-				const { container } = renderProgressIndicator(
-					createList('blocked', { href: '#' })
-				);
-				expect(container).toMatchSnapshot();
-			});
-			it('renders active button', () => {
-				const { container } = renderProgressIndicator(
-					createList('blocked', { onClick: console.log })
-				);
-				expect(container).toMatchSnapshot();
-			});
-		});
+		const activeStatusWhiteList: Array<ProgressIndicatorItemStatus> = [
```

Comment:
Nitpick: Whitelist and Blacklist are one word, so shouldn't have a a capital L. I wonder if we're allowed to use these terms though (since they can have negative connotations)?

Reply:
updated with less problematic names